model-optimizer/mo/front/extractor.py

```diff
@@ -132,14 +132,16 @@ def attr_getter(node: Node, name: str):
 
 
 def bool_to_str(node: Node, attr: str):
-    # Function converts 0/1 or bool False/True values to str 'false'/'true' which need to appear in IR
+    # Function converts 0/1 or bool False/True or '0'/'1' values to str 'false'/'true' which need to appear in IR
     attribute_name = node.soft_get(attr, None)
     if attribute_name is None:
         return None
     if isinstance(attribute_name, bool):
         return str(attribute_name).lower()
     elif attribute_name in [0, 1]:
         return str(bool(attribute_name)).lower()
+    elif attribute_name in ['0', '1']:
+        return str(bool(int(attribute_name))).lower()
```

Comment:
May be just return `attribute_name` in this case? What is the point to convert string to int then to bool and then back to string?

Also, let's rename the variable to `attribute_value`.

Reply:
Please, ignore the comment